base/src/main/scala/co/uproot/abandon/Process.scala

```diff
@@ -7,8 +7,8 @@ import java.io.FileNotFoundException
 
 case class AppState(accState: AccountState, accDeclarations: Seq[AccountDeclaration])
 
-class TxnGroup(
-  _children: Seq[DetailedTransaction],
+class PostingGroup(
```

Comment:
Can you call it `PostGroup` instead of `PostingGroup`?

The term `Posting` might confuse someone.